h/migrations/versions/ec5c9a864c76_migrate_denied_spam.py

```diff
@@ -0,0 +1,55 @@
+"""
+Migrate DENIED -> SPAM.
+
+At the time of this migration all annotations with a moderation_status
+have "DENIED" coming from the hide feature.
+
+Update those rows and the moderation_log ones to use SPAM instead.
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "ec5c9a864c76"
+down_revision = "81a5daab8bfc"
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+    result = conn.execute(
+        sa.text(
+            """
+        WITH candidate_annotations as (
+            -- These are all the old annotations hidden by the hide feature
+            select annotation_id from annotation_moderation
```

Comment:
This is much faster that going over the 70M annos, just look at the ones that we know have moderation status changes based on the old and the new tables.